proparse/src/main/java/org/prorefactor/proparse/antlr4/Lexer.java

```diff
@@ -290,6 +295,8 @@ ProToken getAmpIfDefArg() throws IOException {
    * lexer. I don't think there's any way to get whitespace into a macro name either.
    */
   private ProToken ampIfDefArg() throws IOException {
+    LOGGER.trace("Entering ampIfDefArg()");
+
     loop : for (;;) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS135)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor the code to remove this label and the need for it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ALabelsShouldNotBeUsedCheck)